administrator/components/com_installer/View/Updatesites/Html.php

```diff
@@ -72,6 +72,11 @@ protected function addToolbar()
 	{
 		$canDo = ContentHelper::getActions('com_installer');
 
+		if (($canDo->get('core.edit')) || ($canDo->get('core.edit.own')))
```

Comment:
Why duplicate braces? And i don't think you should check for core.edit.own (Edit own updatesites? Not sure, if you want to go this way you need at least check if owner matches which is imo not working..) Probably better checking for core.manage